vault/sources/Vault.move

```diff
@@ -89,13 +166,13 @@ module titusvaults::Vault {
                 deposits: smart_table::new(),
                 shares: smart_table::new(),
                 rounds: smart_table::new(),
-                total_vaults: 0,
+                total_rounds: 0
             });
             move_to(_host, Vault<VaultT, AssetT> {
                 coin_store: coin::zero(),
                 creation_time: timestamp::now_microseconds(),
                 creation_round: current_round.round,
-                total_shares: 0,
+                total_shares: 0
             });
         };
     }
```

Comment:
In deposit function, accept a round ID parameter and deposit funds into the round ID only.